Formula/global.rb

```diff
@@ -27,8 +28,12 @@ class Global < Formula
     depends_on "libtool" => :build
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
```

Comment:
Looks like this picked up linkage with `libtool`. It would be nice if we could avoid that, but otherwise this is now a runtime dependency.